src/masternodes/mn_rpc.cpp

```diff
@@ -551,8 +551,12 @@ UniValue setgov(const JSONRPCRequest& request) {
                 const auto attrMap = attributes->GetAttributesMap();
                 for (const auto& [key, value] : attrMap) {
                     if (const auto attrV0 = std::get_if<CDataStructureV0>(&key)) {
-                        if (attrV0->type == AttributeTypes::Consortium && (attrV0->typeId == 0 || pcustomcsview->GetLoanTokenByID({attrV0->typeId}))) {
-                            throw JSONRPCError(RPC_INVALID_REQUEST, "Cannot set consortium on DFI or loan tokens");
+                        DCT_ID tokenID{attrV0->typeId};
+                        if (attrV0->type == AttributeTypes::Consortium &&
+                            (attrV0->typeId == 0 ||
+                             !pcustomcsview->GetToken(tokenID)->IsDAT() ||
```

Comment:
Do not use `->IsDAT()` unless you know a valid object has been returned or we will segfault.